demo/long_video_demo.py

```diff
@@ -79,20 +83,20 @@ def show_results_video(result_queue, text_info, thr, msg, frame, video_writer):
             location = (0, 40 + i * 20)
             text = selected_label + ': ' + str(round(score, 2))
             text_info[location] = text
-            cv2.putText(frame, text, location, FONTFACE, FONTSCALE, FONTCOLOR,
+            cv2.putText(fr, text, location, FONTFACE, FONTSCALE, clr,
                         THICKNESS, LINETYPE)
     elif len(text_info):
         for location, text in text_info.items():
-            cv2.putText(frame, text, location, FONTFACE, FONTSCALE, FONTCOLOR,
+            cv2.putText(fr, text, location, FONTFACE, FONTSCALE, clr,
                         THICKNESS, LINETYPE)
     else:
-        cv2.putText(frame, msg, (0, 40), FONTFACE, FONTSCALE, MSGCOLOR,
-                    THICKNESS, LINETYPE)
-    video_writer.write(frame)
+        cv2.putText(fr, msg, (0, 40), FONTFACE, FONTSCALE, clr, THICKNESS,
+                    LINETYPE)
+    v_writer.write(fr)
     return text_info
 
 
-def get_results_json(result_queue, text_info, thr, msg, ind, out_json):
+def get_results_json(result_queue, text_info, msg, thr, ind, out_json):
```

Comment:
It is not good to change the argument position

Reply:
I updated accordingly. But why isn't this one good btw?